src/test/scala/chiselTests/util/PipeSpec.scala

```diff
@@ -1,7 +1,7 @@
-package chiselTests.util
-
 // SPDX-License-Identifier: Apache-2.0
 
+package chiselTests.util
+
```

Comment:
this change is unrelated but I opened this file to copy it for making the `PriorityMuxSpec` and noticed the license comment was in the wrong spot.